tests/core/pyspec/eth2spec/test/electra/epoch_processing/test_apply_pending_deposit.py

```diff
@@ -0,0 +1,138 @@
+from eth2spec.test.context import (
+    spec_state_test,
+    with_electra_and_later,
+)
+from eth2spec.test.helpers.keys import privkeys, pubkeys
+from tests.core.pyspec.eth2spec.test.helpers.deposits import build_deposit_data
+from eth2spec.test.helpers.state import next_epoch_via_block
+
+
+@with_electra_and_later
+@spec_state_test
+def test_apply_pending_deposit_add_validator_to_registry(spec, state):
+    amount = 100
```

Comment:
```suggestion
    amount = spec.MIN_ACTIVATION_BALANCE
```

Maybe use this or another constant to avoid using magic numbers whenever it is possible?